.github/workflows/main.yaml

```diff
@@ -16,68 +16,42 @@ jobs:
         working-directory: ${{ matrix.ubuntu-version }}
     steps:
       - uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: linux/amd64,linux/arm64/v8
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
       - name: Extract targets
         run: |
-          TAG_MINIMAL=$(cat TAG_MINIMAL)
-          c="$(../tag_exists ubuntu-minimal $TAG_MINIMAL)"
-          if [ "$c" = ng ]; then
-              echo "update ubuntu-minimal:$TAG_MINIMAL"
-              echo "yes!" > NEED_UPDATE_MINIMAL
-          fi
           TAG=$(cat TAG)
-          for image in ubuntu ubuntu-dev ubuntu-debug; do
+          for image in ubuntu-minimal ubuntu ubuntu-dev ubuntu-debug; do
```

Comment:
This will change the tag names of our `quay.io/cybozu/ubuntu-minimal` images.
Currently we get `quay.io/cybozu/ubuntu-minimal:focal-20220801`, where `focal-20220801` is used by the official Ubuntu image.
By merging this PR, we'll get `quay.io/cybozu/ubuntu-minimal:20.04.20220926`, where `20.04.20220926` is the contents of the the `TAG` file.

This change needs a prior agreement with the users of the `ubuntu-minimal` images, though I suppose the Neco team is the only user.
And more importantly, if we change the tag names of `ubuntu-minimal`, it would be necessary to modify `18.04/ubuntu/Dockerfile` and `20.04/ubuntu/Dockerfile` to use `ubuntu-minimal:${TAG}` instead of `ubuntu-minimal:${TAG_MINIMAL}`.

Reply:
Thank you. I fixed to set `TAG_MINIMAL` to the tag name for the `quay.io/cybozu/ubuntu-minimal` image.